config/routes.rb

```diff
@@ -2299,6 +2299,35 @@
         save_post
     },
 
+    :storage_resource   => {
+      :get  => %w[
+        discover
+        download_data
+        download_summary_pdf
+        protect
+        show
+        show_list
+        tagging_edit
+      ],
+        :post => %w[
+          button
+          listnav_search_selected
+          protect
+          quick_search
+          show
+          show_list
+          tagging_edit
+          tl_chooser
```

Comment:
@llivne can you please re-visit these routes added here, looks like you don't need most of these.

Reply:
@h-kataria I removed all the unused routes